### ISSUE TITLE
•	Coga Suggested  REQ (Requirement ) Allow search through a page, whole document, a group of documents (such as a project or directory) 

### ISSUE THREAD

**Issue by jasonjgw** [w3c/ctaur#14]
Comment from COGA Task Force, via Scott's summary.

**Comment by jasonjgw** [comment]
On the surface, at least, this proposal seems to be beyond the scope of the current project, as it does not relate to collaborative functionality specifically. It could apply to any document or code editing environment whatsoever, including those that lack collaboration support.

One could reformulate it as a requirement with respect to searching through different revisions of content in a collaborative tool that supports revision control.

Also, it isn't clear what the proposed requirement is, or how to specify the accessibility-related need that it is meant to satisfy.

For example, it could be interpreted as requiring the application to enable the user to select the scope of a search (e.g., search the current file or document, all files in a storage directory, only the current revision, all revision, the current page in a word processor file, etc.). However, the number of possibilities grows quickly, especially in a collaborative setting.

In summary: it isn't clear what has been asked for here, what accessibility-related rationale should be given, or how to apply it in the case of revision-controlled content.

**Comment by sehollier** [comment]
I think where this could be useful is around version control, where it is very difficult in collaboration tools to identify different versions and if one person comes in and changes things, it is difficult to revert back to a previous version so from a cognitive disability perspective, I can see merit in having an easy to monitor/search version control feature if that is what is intended here.

**Comment by JaninaSajka** [comment]
Added version++ as the notion is worth addressing when we can define the ask more clearly.
